### PR TITLE
frontend: Add null check to ChannelList

### PR DESCRIPTION
**Pull request by yolossn**
Fixes: https://github.com/kinvolk/nebraska/issues/318

### CODE REVIEW

File: frontend/src/js/components/Channels/List.react.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ function ChannelList(props) {
 
   function getChannelsPerArch() {
     const perArch = {};
+
+    // If application doesn't have any channel return empty object.
+    if (application.channels === null){
```
Comment (knrt10) [inline frontend/src/js/components/Channels/List.react.js:33]:
```suggestion
    if (application.channels === null) {
```

**Review by knrt10 — CHANGES_REQUESTED** [review]
Small linting NIT

**Comment by knrt10** [comment]
@yolossn can you please sign the commit with your kinvolk email? Thanks

**Comment by yolossn** [comment]
> @yolossn can you please sign the commit with your kinvolk email? Thanks

Done

**Review by ashu8912 — APPROVED** [review]
Tested and LGTM

**Pull request merged**